README.md

```diff
@@ -1,6 +1,10 @@
 # mod_cloudflare for Apache #
 Copyright CloudFlare Inc. 2016
 
+## Deprecation notice ##
+
+Cloudflare no longer updates and supports mod_cloudflare, starting with versions Debian 9 and Ubuntu 18.04 LTS of the Linux operating system. We now support [mod_remoteip](https://support.cloudflare.com/hc/articles/200170786#C5XWe97z77b3XZV) for customers using Apache web servers.
```

Comment:
Nit: maybe better to reword as "We now _recommend_" given that we don't own or directly support mod_remoteip but it can be used instead of mod_cloudflare.

Otherwise lgtm